task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +318,19 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let str = '';
+    let sum = 0;
+    str += num;
+
+    while(str.length > 1) {
+
+        for(var i = 0; i < str.length; i++) {
+            sum += Number(str[i]);
+        }
+        str = String(sum);
+        sum = 0;
+    }   
+    return Number(str);
```

Comment:
You can do it easier also in 3 rows :)
P.S. you can use getDigitalRoot() once again inside of the function